README.md

```diff
@@ -1,19 +1,39 @@
+```
+            _   _       _
+ _ __   ___| |_| | __ _| |__
+| '_ \ / _ \ __| |/ _' | '_ \
+| | | |  __/ |_| | (_| | |_) |
+|_| |_|\___|\__|_|\__,_|_.__/
+```
+
 # bp2022_netlab
 
+netlab helps you to quickly generate configuration files for network experiment softwares such as [CORE](http://coreemu.github.io/core/), [coreemu-lab](https://github.com/gh0st42/coreemu-lab) and [The ONE](https://github.com/akeranen/the-one) using a single TOML file. Its aim is to make your work easier, thus it handles annoying tasks such as generating IP addresses, movement patterns (using [BonnMotion](https://sys.cs.uos.de/bonnmotion/)) and multiple parameterized runs.
+
+netlab was developed by a group of five students as part of a bachelor internship of the [department of computer science](https://www.informatik.tu-darmstadt.de/fb20/index.en.jsp) at [TU Darmstadt](https://www.tu-darmstadt.de/index.en.jsp) in summer term 2023.
```

Comment:
I would specify this more not just conputer science of tu darmstadt. we do this in the working group Software Technology Group